### DISCUSSION TITLE
Using `date` as default date field rather than created

### DISCUSSION THREAD

**Discussion by rufuspollock** [flowershow/flowershow#416] [General and Help]
As per title: this is discussion about default date related field on pages and blog. And what, if any, other date fields to have.

I note that in #326 there is mention that hugo and nextjs-template both use `date.

I think we should move to this too.

As bonus i think we could allow having `created` too. And that in our contentlayer we set `date` to created if `created` is present and `date` is not (backwards compatibility). (I'm not sure if this is possible actually with contentlayer ...).

**Comment by rufuspollock** [comment]
@olayway wdyt?

**Reply by olayway** [reply]
> And that in our contentlayer we set `date` to created if `created` is present and `date` is not (backwards compatibility). (I'm not sure if this is possible actually with contentlayer ...).

@rufuspollock yes, it's possible and very easy to implement. We can make use of contentlayer's computed fields to facilitate that.
And since `date` seems to be prevalent, I'd support it - like you mentioned - with backward compatibility for `created` for now. We can remove the support for `created` in the future.
